packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.html

```diff
@@ -4,8 +4,8 @@
       <div class="slds-col slds-truncate slds-col_bump-right breadcrumb-manager__column">
         <ul>
           <template for:each={facetBreadcrumbValues} for:item="breadcrumb">
-            <li key={breadcrumb.facetId} class="slds-list_horizontal breadcrumb-manager__line">
-              <span class="slds-col breadcrumb-manager__field-name">{breadcrumb.field}{labels.colon} </span>
+            <li key={breadcrumb.facetId} class="slds-list_horizontal facetBreadcrumb__list breadcrumb-manager__line">
```

Comment:
To follow the BEM convention, the class name should be something like that `breadcrumb-manager__facet-list`.